state/runtime/tracer/structtracer/tracer.go

```diff
@@ -55,17 +55,22 @@ type StructTracer struct {
 	consumedGas uint64
 	output      []byte
 	err         error
-	storage     map[types.Address]map[types.Hash]types.Hash
 
-	currentMemory []byte
-	currentStack  []*big.Int
+	storage       []map[types.Address]map[types.Hash]types.Hash
+	currentMemory []([]byte)
```

Comment:
why using parenthesis, why not just 	`currentMemory [][]byte ` ?

Reply:
It was just for readability I guess, I'm ok with both forms